### PR TITLE
Improve debug counter enums

### PR DESCRIPTION
**Pull request by daall**
* Add START and END attributes for drop reason enums
* Add additional port stats for debug counters
* Add additional switch stats for debug counters

### CODE REVIEW

File: inc/saidebugcounter.h [hunk 1]
```diff
@@ -81,6 +81,9 @@ typedef enum _sai_debug_counter_bind_method_t
  */
 typedef enum _sai_in_drop_reason_t
 {
+    /** Start of in drop reasons */
+    SAI_IN_DROP_REASON_START,
```
Comment (kcudnik) [inline inc/saidebugcounter.h]: sai_in_drop_reason_t: first enum should start with zero
assign this value to the first item, check _sai_hostif_trap_type_t

File: inc/saidebugcounter.h [hunk 2]
```diff
@@ -310,6 +319,9 @@ typedef enum _sai_in_drop_reason_t
  */
 typedef enum _sai_out_drop_reason_t
 {
+    /** Start of out drop reasons */
+    SAI_OUT_DROP_REASON_START,
```
Comment (kcudnik) [inline inc/saidebugcounter.h]: same here

File: inc/saiport.h [hunk 3]
```diff
@@ -1741,12 +1741,54 @@ typedef enum _sai_port_stat_t
...
+
+    /** Get in port packet drops configured by debug counter API at index 5 */
+    SAI_PORT_STAT_IN_CONFIGURED_DROP_REASONS_5_DROPPED_PKTS,
+
+    /** Get in port packet drops configured by debug counter API at index 6 */
+    SAI_PORT_STAT_IN_CONFIGURED_DROP_REASONS_6_DROPPED_PKTS,
+
+    /** Get in port packet drops configured by debug counter API at index 7 */
+    SAI_PORT_STAT_IN_CONFIGURED_DROP_REASONS_7_DROPPED_PKTS,
+
```
Comment (fengyingling) [inline inc/saiport.h:1764]: What exactly do these attributes mean? It would be best if there are some examples -_-？

Comment (fengyingling) [inline inc/saiport.h:1764]: What exactly do these attributes mean? It would be best if there are some examples -_-？

**Pull request merged**